eq-author-api/src/businessLogic/onSectionDeleted.js

```diff
@@ -2,9 +2,12 @@ const onPageDeleted = require("../../src/businessLogic/onPageDeleted");
 const { getPages } = require("../../schema/resolvers/utils");
 
 const onSectionDeleted = (ctx, removedSection) => {
-  removedSection.pages.forEach(page => {
-    onPageDeleted(ctx, removedSection, page);
+  removedSection.folders.forEach(folder => {
+    folder.pages.forEach(page => {
+      onPageDeleted(ctx, removedSection, page);
+    });
```

Comment:
Some duplication here because `onFolderDeleted` already handles calling `onPageDeleted` for all of the folder's child pages. Could delegate to that instead:

```suggestion
  removedSection.folders.forEach(folder => {
    onFolderDeleted(ctx, folder);
```